scripts/TPI.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+import scipy.optimize as opt
+
```

Comment:
I would recommend instead of defining FOCs again here:
import firm
import households as hh
import aggregates as agg